apps/fluent-tester/src/TestComponents/Overflow/OverflowTest.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import type { ViewStyle } from 'react-native';
+
+import { ButtonV1 as Button } from '@fluentui-react-native/button';
+import { Divider } from '@fluentui-react-native/divider';
+import { Menu, MenuTrigger, MenuPopover, MenuItem } from '@fluentui-react-native/menu';
+import { Overflow, OverflowItem, useOverflowMenu } from '@fluentui-react-native/overflow';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+
+import MoreHorizontalIcon from './MoreHorizontalFilled.svg';
+import { Test } from '../Test';
+
+const items = ['a', 'b', 'c', 'd', 'e'];
+const itemLabels = {
+  a: 'Item A',
+  b: 'Item B Item B Item B',
+  c: 'Item C Item C',
+  d: 'Item D',
+  e: 'Item E Item E Item E Item E',
+};
+
+const containerStyle: ViewStyle = {
+  paddingVertical: 4,
+};
+
+interface OverflowMenuProps {
+  onItemClick: (id: string) => void;
+}
+
+function OverflowMenu(props: OverflowMenuProps) {
+  const { showMenu, menuItems, menuRef, onLayout } = useOverflowMenu();
+  if (showMenu) {
+    return (
+      <Menu>
+        <MenuTrigger>
+          <Button
+            onLayout={onLayout}
```

Comment:
I would add an accessibilityLabel to this button since the accessibiltyLabel of a focusable item should not be null. Could be something like "More options".